src/sql/parts/notebook/notebook.component.ts

```diff
@@ -157,6 +160,11 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 	}
 
+	//Saves scrollTop value on scroll change
+	public scrollHandler(event: Event){
+		this._scrollTop = event.srcElement.scrollTop;
```

Comment:
Does this work when a notebook and a query editor are side by side in split view?